test/BinaryParse.cpp

```diff
@@ -384,25 +316,14 @@ TEST_F(BinaryParseTest, EarlyReturnWithOnePassingCallback) {
       "%1 = OpTypeVoid "
       "%2 = OpTypeInt 32 1 "
       "%3 = OpTypeFloat 32");
-
-  num_passing_callbacks_ = 1;
-  fail_code_ = SPV_REQUESTED_TERMINATION;
-  spv_diagnostic diagnostic = nullptr;
-  EXPECT_EQ(SPV_REQUESTED_TERMINATION,
-            spvBinaryParse(context, this, binary.data(), binary.size(), Header,
-                           Instruction, &diagnostic));
+  InSequence calls_expected_in_specific_order;
+  EXPECT_HEADER(4).WillOnce(Return(SPV_REQUESTED_TERMINATION));
```

Comment:
This test is OnePassingCallback, so the header call should return SPV_SUCCESS, and we should get exactly one call to the instruction callback, which should have the correct contents for the  OpTypeVoid, and then itself should return SPV_REQUESTED_TERMINATION.